BUILD/familiars/drop.dat

```diff
@@ -2,6 +2,8 @@
 #
 # First we grab up to small amount of various specific drops.
```

Comment:
Make sure to run assemble.sh, located in autoscend\BUILD,  to recreate the .txt files. 

Guessing you did this but didn't include in the PR? The .txt files are what the script uses

Reply:
Ah, I didn't realize you had to do this and check the change in, I thought some build step would handle this. Will do.

(When testing I applied the change directly to the built files in my mafia-installed version so I didn't run into this.)